examples/base_difference_mapcubes.py

```diff
@@ -0,0 +1,41 @@
+"""
+========================================
+Base Difference in Mapcubes
+========================================
+
+"""
+############################################################################
+
+# Importing Libraries
+import numpy as np
+import matplotlib.pyplot as plt
+from datetime import date
+import matplotlib.colors as colors
+
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+# Forming the maps
+aia1 = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE)
+aia2 = sunpy.map.Map(AIA_193_CUTOUT02_IMAGE)
+aia3 = sunpy.map.Map(AIA_193_CUTOUT03_IMAGE)
```

Comment:
Rather than individually constructing these three maps, I would just construct one mapcube:
```
sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE, cube=True)
```

Reply:
This doesn't work: _ValueError: MapCube expects pre-constructed map objects._

Reply:
This works @wafels.
I guess instead of aia1, aia2,aia3, we can use 

`aiamap = sunpy.map.Map(AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE, cube=True)`

and access it through `aiamap[0]` and `aiamap[1]`...